tasks/Fedora.yml

```diff
@@ -28,12 +28,13 @@
     - upgrade
 
 - name: Check System Reboot
-  command: tracer
+  command: needs-restarting -r
```

Comment:
tracer method worked when I tested locally but on travis build hangs, I switched to `dnf-utils` and now is working as expected like centos

Reply:
looks awesome, thanks.